__tests__/app.test.js

```diff
@@ -74,4 +83,55 @@ describe("GET /api/articles/:article_id", () => {
         expect(response.body.msg).toBe("Bad request");
       });
   });
+// });
+
+describe("GET /api/articles", () => {
+  test("GET:200 status responds with an array of articles with expected properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((res) => {
+        const article = res.body.articles;
+        article.forEach((article) => {
+          expect(typeof article.article_id).toBe("number");
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.topic).toBe("string");
+          expect(typeof article.created_at).toBe("string");
+          expect(typeof article.votes).toBe("number");
+          expect(typeof article.article_img_url).toBe("string");
+        });
+      });
+  });
+  test("GET:200 status responds with an array of articles with expected properties along with comment_count", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((res) => {
+        const article = res.body.articles;
+        expect(article.hasOwnProperty("article")).toBe(false);
+        article.forEach((article) => {
+          expect(typeof article.article_id).toBe("number");
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.topic).toBe("string");
+          expect(typeof article.created_at).toBe("string");
+          expect(typeof article.votes).toBe("number");
+          expect(typeof article.article_img_url).toBe("string");
+          expect(typeof article.comment_count).toBe("string");
```

Comment:
It would be ideal for comment_count to be a number, check out my comment further down for a tip on how to do this